### PR TITLE
fix: dark mode icons for Riverside and Sirius Video

### LINKED ISSUE #8966: [CAL-1721] Dark mode icons for Riverside and Sirius Video aren't visible
image

<sub>[CAL-1721](https://linear.app/calcom/issue/CAL-1721/dark-mode-icons-for-riverside-and-sirius-video-arent-visible)</sub>

### PR DESCRIPTION
**Pull request by CoderGhost37**
Fixes #8966

On installed apps menu:
239249345-bece772b-2a64-4808-a03b-fb652c1682af

On event preview:
239471456-cc841cd4-667a-4bfb-985a-256f00b8b775

On edit event form (location field):
239471541-8331c096-6949-43d1-adc9-3552a882609e

### CODE REVIEW

File: yarn.lock
Comment (hariombalhara) [inline yarn.lock:1]: Let's revert this change.

File: packages/features/bookings/components/event-meta/AvailableEventLocations.tsx [hunk 1]
```diff
@@ -22,8 +22,8 @@ function RenderIcon({
     <img
       src={eventLocationType.iconUrl}
       className={classNames(
-        "me-[10px]  h-4 w-4 opacity-70 dark:opacity-100",
-        !eventLocationType.iconUrl?.startsWith("/app-store") ? "dark:invert-[.65]" : "",
+        "me-[10px] h-4 w-4 opacity-70 dark:opacity-100",
+        eventLocationType.iconUrl?.includes("-dark") ? "dark:invert-[.65]" : "",
         !eventLocationType.iconUrl?.startsWith("/app-store") && isTooltip && "invert"
```
Comment (hariombalhara) [inline packages/features/bookings/components/event-meta/AvailableEventLocations.tsx:27]: Do we not want to fix this as well? It is also checking for `/app-store`

**Review by hariombalhara — DISMISSED** [review]
@CoderGhost37 Thanks for the PR 🙏
In this [image](https://user-images.githubusercontent.com/92794376/265251924-c11b256c-6fd1-4b06-aa96-7d0abfcf91fe.png) that you shared, I can see that In Person Attendee and Organizer address icons aren't visible in dark mode.

Also, please add screenshots of how the icons look in light mode.

File: yarn.lock
Comment (CoderGhost37) [reply yarn.lock:1]: There is no change in the yarn.lock file I don't know why its showing this as a change

File: packages/features/bookings/components/event-meta/AvailableEventLocations.tsx (hunk 1 above)
Comment (CoderGhost37) [reply packages/features/bookings/components/event-meta/AvailableEventLocations.tsx:27]: Done

**Review by PeerRich — APPROVED** [review]
